features/integration/applications.feature

```diff
@@ -125,15 +126,83 @@ Feature: Applications
+      And I sign and submit the transaction, saving the txid. If there is an error it is "".
+      And I wait for the transaction to be confirmed.
+      Then according to "algod", the contents of the box with name "str:foo bar" in the current application should be "YmF6IHF1eAAAAAAAAAAAAAAAAAAAAAAA". If there is an error it is "".
+      And I sleep for 500 milliseconds for indexer to digest things down.
+      And according to "indexer", the contents of the box with name "str:foo bar" in the current application should be "YmF6IHF1eAAAAAAAAAAAAAAAAAAAAAAA". If there is an error it is "".
+
+      # full check confirmed by both algod and indexer
+      Then according to "algod", the current application should have the following boxes "Zm9vIGJhcg==:APj/IA==:bmFtZQ==:MTE0NTE0".
+      And according to "algod", by parameter max 2, the current application should have 2 boxes.
+      And according to "algod", by parameter max 6, the current application should have 4 boxes.
+
+      And I sleep for 500 milliseconds for indexer to digest things down.
+      And according to "indexer", the current application should have the following boxes "Zm9vIGJhcg==:APj/IA==:bmFtZQ==:MTE0NTE0".
+      And according to "indexer", by parameter max 2, the current application should have 2 boxes.
+      And according to "indexer", by parameter max 6, the current application should have 4 boxes.
+
+      # delete one box
+      Then I build an application transaction with the transient account, the current application, suggested params, operation "call", approval-program "", clear-program "", global-bytes 0, global-ints 0, local-bytes 0, local-ints 0, app-args "str:delete,str:name", foreign-apps "", foreign-assets "", app-accounts "", extra-pages 0, boxes "0,str:name"
+      And I sign and submit the transaction, saving the txid. If there is an error it is "".
+      And I wait for the transaction to be confirmed.
+      And according to "algod", the current application should have the following boxes "Zm9vIGJhcg==:APj/IA==:MTE0NTE0".
+
+      # move to indexer testing steps
+      And I sleep for 500 milliseconds for indexer to digest things down.
+      And according to "indexer", the contents of the box with name "str:name" in the current application should be "". If there is an error it is "no application boxes found".
+      And according to "indexer", the contents of the box with name "b64:APj/IA==" in the current application should be "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA". If there is an error it is "".
+      And according to "indexer", the current application should have the following boxes "Zm9vIGJhcg==:APj/IA==:MTE0NTE0".
+      And according to "indexer", by parameter max 2, the current application should have 2 boxes.
```

Comment:
```suggestion
      And according to "indexer", with 2 being the parameter that limits results, the current application should have 2 boxes.
```
The reason for the suggested change is that the "max" terminology makes sense for `algod` but not `indexer`. If the step i intended for use by indexer only, you could just change "max" to "limit".

Reply:
Take this suggestion with a grain of salt...not at all a big deal.

Reply:
put this on the top of my stack tomorrow

Reply:
just adapted your suggestion.